hail/python/hail/experimental/vcf_combiner.py

```diff
@@ -137,6 +132,7 @@ def lgt_to_gt(lgt, la):
 
 
 def summarize(mt):
+    from . import densify
```

Comment:
this isn't good style - if importing at the top would lead to circular imports, just use the fully qualified name `hl.experimental.densify`